### ISSUE TITLE
Where to specify app contextual data in ortb2 object? 

### ISSUE THREAD

**Issue by patmmccann** [prebid/Prebid.js#7415]
## Type of issue
Standards

## Description
#6057 proposes where to put content taxonomies. If they are contextual they go in site.content.data, but the site field is not valid openRTB when the app field exists, so we would propose app.content.data with the structure mirroring that defined in IAB SDA proposal and in #6057 

Separately, there appears to be a need for a segtax 7 for app focused content data at https://github.com/InteractiveAdvertisingBureau/AdCOM/blob/master/AdCOM%20v1.0%20FINAL.md#list_categorytaxonomies ; or additions to the segtax 6 = content 2.2 that are app focused such as those proposed at https://docs.google.com/spreadsheets/d/1T4P7DfvAPLvwmNC_mSwfHzaW57M2uy__/edit?usp=sharing&ouid=105283563856401027398&rtpof=true&sd=true. 

If data provider wanted to specify some data from existing content taxonomy 6 and future app context taxonomy 7, the object could look like this: 

	ortb2: {
		app: {
			name: "myappname",
			keywords: "power tools, drills",
			content: {
				data: [{
					name: "www.dataprovider1.com",
					ext: {
						"segtax": 6
					},
					segment: [{
						id: "687"
					}, {
						id: "123"
					}]
				}],
				  data: [{
					name: "www.dataprovider1.com",
					ext: {
						"segtax": 7
					},
					segment: [{
						id: "456"
					}, {
						id: "789"
					}]
				}]
			}
		}

**Comment by patmmccann** [comment]
Ty for the correction. The taxonomy committee uses this board, but perhaps you are correct this would get more attention on the SDK board. I am not familiar with that.

**Comment by patmmccann** [comment]
tl;dr: app.content.data should be documented somewhere as following the same structure as site.content.data

**Comment by patmmccann** [comment]
Closing with submission of https://github.com/prebid/prebid.github.io/pull/3299
